.travis.yml

```diff
@@ -8,14 +8,38 @@ services:
 env:
   - DATABASE_URI=postgres://postgres:@localhost:5432/test
 
-before_script:
-  - psql -c 'create database test;' -U postgres
```

Comment:
I think we still need this, `lab-flask-bdd` just doesn't use postgres, but we do.